lib/better_errors/error_page.rb

```diff
@@ -70,7 +71,7 @@ def first_frame
       application_frames.first || backtrace_frames.first
     end
 
-  private
+    private
```

Comment:
Not a big deal, but the indentation of this is based on the "standard" Ruby, which Rails doesn't follow. So for Rails projects `private` will be at the same indentation level as the method definitions, but in non-Rails projects, it will generally be outdented one level, inline with the `class` line.

Reply:
Got it. I had let the auto indent take care of this. I guess it's using the rails and not the ruby standard for indentation. I'll keep this in mind when running the autoindenter.